### DISCUSSION TITLE
datalisp

### DISCUSSION THREAD

**Discussion by zx9w** [subconsciousnetwork/noosphere#41] [Show and tell]
I am doing something similar to you guys over at: https://sr.ht/~ilmu/tala.saman/

**Comment by cdata** [comment]
Thanks for sharing @zx9w !

It looks like you might be early on your journey (as we are). Maybe there are ideas we can share across projects, or even contributions. Would you like to offer any wisdom for us, coming from your experience designing a similar project?

**Reply by zx9w** [reply]
Hi, sorry for the late reply! I am not really a github user.

So I have made a small site: datalisp.is and there is a matrix chat https://matrix.to/#/#datalisp:matrix.org

I believe we should encode the web as canonical S-expressions. In one sentence I'd say something like "the noosphere is a subspace of syntactically correct cxps".

The reason we pay taxes and the reason for climate change; both can be attributed to the tragedy of the commons. If we are all honestly looking for ways to align incentives then we should start by just accepting the simplest sufficient premise for having that discussion.

I'm not yet very productive as a programmer. I really don't like working alone. I would love to have a jitsi or something where we could share ideas.
